VERSION

```diff
@@ -1 +1 @@
-2.2.0
+2.3.0
```

Comment:
Should this be a major version?

Obviously not much has changed in this module, but if other parts of the terraform are based on the older version of the provider this could be a breaking change for someone that's upgrading.